runtime/v2/example/example.go

```diff
@@ -30,11 +29,6 @@ import (
 	"github.com/containerd/containerd/runtime/v2/shim"
 )
 
-var (
-	// check to make sure the *exampleTaskService implements the GRPC API
-	_ = (taskAPI.TaskService)(&exampleTaskService{})
```

Comment:
This is not needed, `newTaskService` function signature already makes sure that `exampleTaskService` implements `TaskService`.